lab11_4/src/main.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { useState } from 'react';
+
+import './style.css';
+
+const ColorPicker = ({initialColor = "#fff"}) => {
+  const [selected, setSelected] = useState(initialColor);
+  const [colorArray, setColorArray] = useState(

```

Comment:
Does this really belong in state?

Reply:
i guess what i needed to use was useRef 👍

Reply:
Isn't the array a calculated value? why ref?